html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <div>Email</div>
+    <label for="email">
+      <input type="text">
+    </label>
+
+    <label for="coding-cool">
+      <div>Why is coding cool?</div>
+    </label>
+    <select name="options" id="question-2">
```

Comment:
The `for` attribute of the `label` element should match the `id` of the `input` you want to associate the `label` to.

Reply:
Can the `name` and `id` for the select element be the same value? I believe the name is pushed to the server and id is used for CSS styling, but I'm not entirely sure.